Add event app

redmyt opened the pull request:
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)


## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [x] @makarman
- [x] @noobhunters
- [x] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue
    Added event model
    Added migration file
    Changed settings file
    Added docstrings

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

redmyt committed:
Add event app

Added event model
Added migration file
Changed settings file
Added docstrings

skramar commented on eventually/event/models.py:
```diff
@@ -0,0 +1,252 @@
+        :type status: integer
+    """
+
+    STATUS_CHOISES = (
```
Choices

redmyt replied:
fixed

skramar commented on eventually/event/views.py:
```diff
@@ -0,0 +1,8 @@
+===========
+"""
+
+# from django.shortcuts import render
```
Do you need commented lines in git repository?

redmyt replied:
These comments occur only for successful travis-pylint verification.

lhalam commented on eventually/event/models.py:
```diff
@@ -0,0 +1,252 @@
+        """
+        Method that converts event object to dictionary.
+
+        :return: dictionary with event's information
```
add example

redmyt replied:
added

lhalam commented on eventually/event/models.py:
```diff
@@ -0,0 +1,252 @@
+            event = Event.objects.get(id=event_id)
+            return event
+        except Event.DoesNotExist:
+            return None
```
maybe `pass`

redmyt replied:
used pass

lhalam commented on eventually/event/models.py:
```diff
@@ -0,0 +1,252 @@
+        event.longitude = longitude
+        event.latitude = latitude
+        event.budget = budget
+        if status:
```
???

redmyt replied:
removed that `if` statement

lhalam commented on eventually/eventually/settings.py:
```diff
@@ -31,6 +31,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'event',
```
your app after djandoApp

redmyt replied:
fixed

lhalam commented on eventually/event/models.py:
```diff
@@ -0,0 +1,268 @@
+            'id': self.id,
+            'name': self.name,
+            'description': self.description,
+            'start_at': self.start_at,
```
convert to timestamp

redmyt replied:
added for all date fields

redmyt committed:
Provide fixes according to reviews

    change settings file
    resolve merge conflict
    fix model fields
    add example for docstrings
